src/core-app-plane/core-app-plane.ts

```diff
@@ -1,48 +1,11 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import * as cdk from 'aws-cdk-lib';
-import { EventBus } from 'aws-cdk-lib/aws-events';
 import * as iam from 'aws-cdk-lib/aws-iam';
 import { Construct } from 'constructs';
 import { BashJobOrchestrator } from './bash-job-orchestrator';
 import { BashJobRunner } from './bash-job-runner';
-import { DestroyPolicySetter } from '../cdk-aspect/destroy-policy-setter';
-import { EventManager, EventMetadata, DetailType, setTemplateDesc } from '../utils';
-
-/**
- * Provides metadata for outgoing events.
- */
-export interface OutgoingEventMetadata {
-  /**
-   * The detailType to set in the outgoing event.
-   */
-  readonly detailType: string;
-
-  /**
-   * The source to set in the outgoing event.
-   *
-   * @default CoreApplicationPlaneProps.applicationPlaneEventSource
-   */
-  readonly source?: string;
-}
-
-/**
- * Provides metadata for incoming events.
- */
-export interface IncomingEventMetadata {
-  /**
-   * The list of detailTypes to listen for in the incoming event.
-   */
-  readonly detailType: string[];
-
-  /**
-   * The list of sources to listen for in the incoming event.
-   *
-   * @default CoreApplicationPlaneProps.controlPlaneEventSource
-   */
-  readonly source?: string[];
-}
+import { EventManager, DetailType, setTemplateDesc } from '../utils';
```

Comment:
```suggestion
import { IEventManager, DetailType, setTemplateDesc } from '../utils';
```

Reply:
Needs both `EventManager` and `IEventManager`